### ISSUE TITLE
Giada Channels represented in two rows

### ISSUE THREAD

**Issue by tomek-szczesny** [monocasual/giada#405]
As I'm pushing my midi overhaul effort, I often think about the future potential uses of this work. One of them is, of course, using a button grid controller to play and stop channels.

Currently channels GUI representation is very wide comparing to their height, which hardly corresponds to any MIDI controller I've ever seen. More than 8 rows of buttons is probably not even available.

If the channel representation could be (optionally or permanently) broken down into two lines, one with its name and another with all that buttons and indicators, id would be far easier to squeeze in 8 columns into a typical modern display.

I'd personally aim for 8 columns on 1280 or 1366 pixels wide display, for comfortable use on older setups. 8 rows and columns is probably the biggest button grid available on the market and I wouldn't worry about bigger setups for now.

**Comment by monocasual** [comment]
I agree channel UI will need some love in the future. This will happen when we'll switch to #86 . For now you can always adjust the column width and the channel height by dragging the empty space between elements. Does it help?

**Comment by tomek-szczesny** [comment]
Well yes, personally I use a 30 inch monitor so I don't mind even the default column widths, but when I take Giada to a jam session on my lappie I'd rather see it with more optimal layout. :)
I don't think this is an urgent request, just something to keep in mind that Giada should be able to fit 8 columns on laptops, without sacrificing channel name visibility.

**Comment by gvnnz** [comment]
We have channel groups now, everything is customizable. closing this.
